frontend/src/pages/org/archived-item-detail/archived-item-detail.ts

```diff
@@ -537,9 +540,14 @@ export class ArchivedItemDetail extends TailwindElement {
   private renderHeader() {
     return html`
       <header class="mb-3 flex flex-col gap-2 border-b pb-3 lg:flex-row">
-        <h1 class="flex w-full min-w-0 text-xl font-semibold leading-8">
-          ${this.renderName()}
-        </h1>
+        <sl-tooltip
+          class="break-all"
+          content="${ifDefined(this.crawl?.name || this.crawl?.firstSeed)}"
```

Comment:
Should this match the behaviour of `this.renderName`? i.e. rendering the crawl id if there's neither a name, first seed, nor seed count

Reply:
That would probably be good for resiliency, though shouldn't happen!